scripts/build_wallon_demands.py

```diff
@@ -0,0 +1,1148 @@
+        'residential urban decentral biomass boiler': ('VAR_FOut', 'combined', [('process_agg', ['Residential urban decentral biomass heater']), ('pypsa_carrier', ['Heat'])]),
+        'residential urban decentral solar thermal': ('VAR_FOut', 'combined', [('process_agg', ['Residential urban decentral solar thermal']), ('pypsa_carrier', ['Heat'])]),
+        'residential urban decentral oil boiler': ('VAR_FOut', 'combined', [('process_agg', ['Residential urban decentral oil heater']), ('pypsa_carrier', ['Heat'])]),
+        #Residential rural
+        'residential rural gas boiler': ('VAR_FOut', 'combined', [('process_agg', ['Residential rural gas heater']), ('pypsa_carrier', ['Heat'])]),
+        'residential rural coal boiler': ('VAR_FOut', 'combined', [('process_agg', ['Residential rural Coal heater']), ('pypsa_carrier', ['Heat'])]),
+        'residential rural electric heater': ('VAR_FOut', 'combined', [('process_agg', ['Residential rural electric heater']), ('pypsa_carrier', ['Heat'])]),
+        'residential rural heat pump': ('VAR_FOut', 'combined', [('process_agg', ['Residential rural Heat pump']), ('pypsa_carrier', ['Heat'])]),
+        'residential rural geothermal': ('VAR_FOut', 'combined', [('process_agg', ['Residential rural geothermal heating']), ('pypsa_carrier', ['Heat'])]),
+        'residential rural biomass boiler': ('VAR_FOut', 'combined', [('process_agg', ['Residential rural biomass heater']), ('pypsa_carrier', ['Heat'])]),
+        'residential rural solar thermal': ('VAR_FOut', 'combined', [('process_agg', ['Residential rural solar thermal']), ('pypsa_carrier', ['Heat'])]),
+        'residential rural oil boiler': ('VAR_FOut', 'combined', [('process_agg', ['Residential rural oil heater']), ('pypsa_carrier', ['Heat'])]),
+        #Commercial
+        'services gas boiler': ('VAR_FOut', 'combined', [('process_agg', ['Commercial gas boiler']), ('pypsa_carrier', ['Heat'])]),
+        'services biomass boiler': ('VAR_FOut', 'combined', [('process_agg', ['Commercial Biomass boiler']), ('pypsa_carrier', ['Heat'])]),
+        'services heat pump': ('VAR_FOut', 'combined', [('process_agg', ['Commercial Heat pump']), ('pypsa_carrier', ['Heat'])]),
+        'services district heating': ('VAR_FOut', 'combined', [('process_agg', ['Commercial Heat Exchanger']), ('pypsa_carrier', ['Heat'])]),
+        'services oil boiler': ('VAR_FOut', 'combined', [('process_agg', ['Commercial Oil boiler']), ('pypsa_carrier', ['Heat'])]),
+        'services geothermal': ('VAR_FOut', 'combined', [('process_agg', ['commercial Geothermal']), ('pypsa_carrier', ['Heat'])]),
+        'services electric heater': ('VAR_FOut', 'combined', [('process_agg', ['Commercial electrical stove']), ('pypsa_carrier', ['Heat'])]),
+        'services solar thermal': ('VAR_FOut', 'combined', [('process_agg', ['Commercial solar thermal']), ('pypsa_carrier', ['Heat'])]),
+    }
```

Comment:
works -- but wouldn't it be easier to revisit if imported from for example a CSV file?